cblox/src/core/tsdf_esdf_submap.cpp

```diff
@@ -9,7 +10,41 @@ void TsdfEsdfSubmap::generateEsdf() {
                                           esdf_map_->getEsdfLayerPtr());
   // Generate the ESDF
   LOG(INFO) << "Generating ESDF from TSDF for submap with ID: " << submap_id_;
+  esdf_integrator.setFullEuclidean(false);
```

Comment:
This is a parameter in the `esdf_integrator_config_`. Taking it from the params has the advantage that it is configurable. Is there any reason that we need this had coded here?